packages/flutter/lib/cupertino.dart

```diff
@@ -8,6 +8,11 @@
 ///
 /// To use, import `package:flutter/cupertino.dart`.
 ///
+/// This library is designed for apps that run on iOS or iPadOS. For apps that
```

Comment:
let's leave iPadOS out or add a caveat since we never invested much in tablet specific UI (which has not many but some noticeable differences)